### PR TITLE
Add support for additional versions of Python

### LINKED ISSUE #1: Support older versions of Python
We currently only test against Python 3.5.

We should be able to support Python 2.7.

### PR DESCRIPTION
**Pull request by alexwlchan**
I added 2.7 support (because I need it for another project), and 3.4 and 3.6, which are the two other major versions of 3.x currently in support. Plus testing in Travis.

Resolves #1.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -9,6 +9,7 @@ commands = {posargs:coverage run -m pytest}
 
 
 [testenv:flake8]
+basepython = python3.5
```
Comment (inglesp) [inline tox.ini:12]: Out of interest, why 3.5 here?

**Review by inglesp — COMMENTED** [review]
All looks great.  Just the one question.

File: tox.ini (hunk 1 above)
Comment (alexwlchan) [reply tox.ini:12]: Totally arbitrary. I think just because 3.5 was the original supported version?

Comment (inglesp) [reply tox.ini:12]: Fair enough!

**Pull request merged**